src/lib/core/components/computations/ComputeJobStatusHook.ts

```diff
@@ -75,7 +80,6 @@ export function useComputeJobStatus(
 
     // Fetch the job status and update state
     async function updateJobStatus() {
-      if (!computePlugin.isConfigurationValid(jobStatusDeps.config)) return;
```

Comment:
okay so the idea is that if the compute config isn't valid, we shouldn't ever even enter the loop. This makes sense